assets/tests/components/propertiesPanel/vehiclePropertiesPanel.test.tsx

```diff
@@ -186,13 +187,15 @@ describe("VehiclePropertiesPanel", () => {
 
   test("Includes invalid bus banner when vehicle is off course", () => {
     render(
-      <VehiclePropertiesPanel
-        selectedVehicle={invalidVehicleFactory.build()}
-        tabMode="status"
-        onChangeTabMode={jest.fn()}
-        onClosePanel={jest.fn()}
-        openMapEnabled={true}
-      />
+      <MemoryRouter initialEntries={["/"]}>
+        <VehiclePropertiesPanel
+          selectedVehicle={invalidVehicleFactory.build()}
+          tabMode="status"
+          onChangeTabMode={jest.fn()}
+          onClosePanel={jest.fn()}
+          openMapEnabled={true}
+        />
+      </MemoryRouter>
     )
```

Comment:
nit(non-blocking): Since you've already made the change, and I've already reviewed it, I don't need this changed; to reduce the diff and whitespace changes, I think using the `wrapper` option or a custom render function would make it easier to review these changes. https://testing-library.com/docs/react-testing-library/setup#custom-render

Reply:
20c57b81